app/index/template.hbs

```diff
@@ -6,12 +6,13 @@
           {{fa-icon "fa-plus-square"}}
         </button>
       {{/if}}
-      <input
+      {{input
         class='new-topic-input'
         type='text'
-        value={{newTopic}}
+        value=newTopic
         placeholder='Suggest a Topic'
-        oninput={{action "newTopicUpdated" value="target.value"}} />
+        key-press='newTopicUpdated'
+        value=target.value }}
```

Comment:
Cool so basically we just need `value=newTopic` and we can get rid of 
        `key-press='newTopicUpdated'
        value=target.value`

I should have made sure I didn't have two values!

Do you mean that I can remove `newTopicUpdated(val) {
      this.set('newTopic', val);
    }` from app/index/controller.js?